packages/battery/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright 2017, the Flutter project authors. All rights reserved.
```

Comment:
This is copied from https://github.com/flutter/plugins/blob/master/LICENSE. Is that one incorrect?